entities/modulecapabilities/backup.go

```diff
@@ -21,27 +21,30 @@ type BackupBackend interface {
 	IsExternal() bool
 	// Name returns backend's name
 	Name() string
-	// HomeDir is the home directory of all backup files
-	HomeDir(backupID string) string
+	// HomeDir is the base storage location of all backup files, which can be a bucket, a directory, etc.
+	HomeDir(backupID, overrideBucket, overridePath string) string
```

Comment:
can we keep the naming consistent and either rename here those extra params to `bucketName` and `bucketPath` or propagate this `override*` naming convention further to the rest of the methods

Reply:
I want to use override everywhere because we already have buckets in the codebase

Reply:
ok, then please override convention everywhere